projects/vdk-heartbeat/src/vdk/internal/heartbeat/config.py

```diff
@@ -102,9 +104,10 @@ def __init__(self, config_file=None):
         )
 
         # The Control Service API URL (http://url/data-jobs) without data-jobs suffix
-        self.control_api_url = self._get_atleast_one_value(
-            "CONTROL_API_URL", "VDK_HEARTBEAT_CONTROL_SERVICE_URL"
-        )
+        # If no value is found, argument will not be passed and default will be used
+        self.control_api_url = self.get_value(
+            "CONTROL_API_URL", is_required=False
+        ) or self.get_value("VDK_HEARTBEAT_CONTROL_SERVICE_URL", is_required=False)
```

Comment:
minor comment: Wouldn't be simpler to add `is_required` argument to `_get_atleast_one_value` ?

Reply:
Yes, good point